lib/transports/websocket.js

```diff
@@ -15,7 +16,7 @@ var debug = require('debug')('engine.io-client:websocket');
  * exposed by `ws` for Node environment.
  */
 
-var WebSocket = typeof window !== 'undefined' ? (window.WebSocket || window.MozWebSocket) : require('ws');
+var WebSocket = isBrowserWebSocket || require('ws');
```

Comment:
I feel `isBrowserWebSocket` should be renamed to `BrowserWebSocket` or something.